src/bom/metadata.rs

```diff
@@ -9,11 +9,11 @@ use crate::{author::Authors, Component, IsEmpty, ToXml};
 
 #[derive(Serialize)]
 pub struct Metadata<'a> {
-    timestamp: DateTime<Utc>,
+    pub timestamp: DateTime<Utc>,
```

Comment:
Made these public for accessibility from other stuff, if they don't need to be let me know (it was in testing something out personally, but I wasn't sure why they weren't public to begin with?)